configs/platforms/osx-10.12-x86_64.rb

```diff
@@ -0,0 +1,14 @@
+platform "osx-10.12-x86_64" do |plat|
+  plat.servicetype 'launchd'
+  plat.servicedir '/Library/LaunchDaemons'
+  plat.codename "sierra"
+
+  plat.provision_with 'curl http://pl-build-tools.delivery.puppetlabs.net/osx/homebrew_sierra.tar.gz | tar -x --strip 1 -C /usr/local -f -'
+  plat.provision_with 'ssh-keyscan github.delivery.puppetlabs.net >> ~/.ssh/known_hosts; /usr/local/bin/brew tap puppetlabs/brew-build-tools gitmirror@github.delivery.puppetlabs.net:puppetlabs-homebrew-build-tools'
```

Comment:
They were likely copied in from a previous definition (10.11 or 10.10) and I believe the original reason was for brevity. They **are** conceptually related (since git pulls in the Homebrew tap over SSH) but this isn't very clear.